lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -299,6 +307,24 @@
   stampTemplate('unoptimized-images-tmpl', document.body);
 }
 
+function deprecationsTest() {
+  stampTemplate('deprecations-tmpl', document.head);
+
+  // Some APIs atm are not triggering Chrome's deprecation messags. https://crbug.com/680832.
+  // try {
+  //   MediaStreamTrack.getSources(); // FAIL
+  // } catch(e) {}
+
+  // console.timeline(); // FAIL
```

Comment:
oh, I meant uncomment these but keep smokehouse expectation at 4 and let chrome eventually fixing the broken deprecations break the test, then update the expectations. But no problem